tests/network_test.cpp

```diff
@@ -629,7 +629,10 @@ TEST_F(NetworkTest, node_pbft_sync) {
   period_data2.transactions.push_back(g_signed_trx_samples[3]);
 
   db1->savePeriodData(period_data2, batch);
-  db1->replaceRewardVotes(votes_for_pbft_blk2, batch);
+  std::cout << "Before VOTE cc" << std::endl;
```

Comment:
remove

Reply:
leftover debugging code. I will remove it